documentation/source/user_section/installation/linux.rst

```diff
@@ -208,20 +193,38 @@ First, save your Docker image:
 
       docker ps -a
 
-3. Save container as new image
+3. If you haven't already, save the container as a new image
 
    .. code:: bash
 
-      docker commit <CONTAINER_ID> <YOUR_NAME>/<DISTROS>:<VERSION>
+      docker commit <CONTAINER_ID> <YOUR_NAME>/ubuntu:ubuntu22.04
+
+Forward X11 server:
+
+.. note::
+
+   The following instructions have been tested with Xorg and xWayland.
+
+   Set up may vary if you are using a different X11 server.
+
+1. Install ``xauth`` and ``xhost`` on the host machine, if not already installed:
+
+   For example on Debian/Ubuntu:
+
+   .. code:: bash
+
+      sudo apt install xauth x11-xserver-utils
+
+2. Permit docker access to the X11 Server
+
+   If hosting container from the local machine:
+
+   .. code:: bash
 
-Create an X11 server for handling display:
+      xhost +local:docker
 
-1. Install XQuartz X11 server.
-2. Check ‘Allow connections from network clientsoption inXQuartz\` settings.
-3. Quit and restart XQuartz.
-4. In XQuartz window xhost + 127.0.0.1
-5. In your other Terminal window, run:
-   ``docker run -ti --rm -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix <CONTAINER_ID>``
+3. In your Terminal window, run:
+   ``docker run -it --rm --privileged -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix <YOUR_NAME>/ubuntu:ubuntu22.04``
```

Comment:
Just for uniformity, it might be good to have this as a `..code ::` directive as well.

Reply:
Also, in this section, it might be good to add an additional step 4 (for testing that GUI forwarding is working):

```suggestion
3. In your Terminal window, run:
   
   .. code:: docker run -it --rm --privileged -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix <YOUR_NAME>/ubuntu:ubuntu22.04``

4. Then, you can check if GUI scripts work by running the following command:

   .. code:: sct_check_dependencies

   If GUI scripts are enabled, you should see all green ``[OK]`` results without any errors.